Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -0,0 +1,65 @@
+import org.opensim.modeling.*;
+import java.io.IOException;
+import static java.nio.file.StandardOpenOption.*;
+import java.nio.file.*;
+
+class TestEditMarkers {
+  public static void main(String[] args) {
+    try {
+        // GUI model loading
+        Model model = new Model("gait10dof18musc_subject01.osim");
+        OpenSimContext context=null;
+        context = new OpenSimContext(model.initSystem(), model);
+        MarkerSet markerset = model.getMarkerSet();
+        // The following code exercises the Marker->Add New  functionality
```

Comment:
> What exactly is the "Add New" callback method?

I mean exactly "what's invoked when the GUI user picks "Add New" context command from the "Markers" node in the navigator." You say it is "essentially" what is invoked, which tells me it is not exactly the same code. 

The approach here seems a bit backwards. The test should call the method you have implemented for "Add New" not a copy of that code. It does not seem optimal to test code here and hope it gets ported in the right way to the right place. Perhaps to satisfy both requirements (to be callable in the GUI and API) is to make it part of OpenSimContext? It seems to me that was one of the purposes of OpenSimContext and its tests.